file-formats/cota/cota-v1.json

```diff
@@ -66,15 +66,12 @@
           "description": "Communication type",
           "type": "string",
           "enum": [
-            "rfc",
             "http"
```

Comment:
I think we had already the discussion whether there should be an enum with only a single value, didn't we?

Technical this won't be necessary from my point of view.

Reply:
Sorry, I missed your comment that it is planned to extend it by "RFC" in the future.